client/src/main/java/edu/colostate/cs/cs414/chesshireCoders/jungleClient/account/AccountHandler.java

```diff
@@ -7,15 +7,17 @@
 import edu.colostate.cs.cs414.chesshireCoders.jungleClient.client.AuthTokenManager;
 import edu.colostate.cs.cs414.chesshireCoders.jungleClient.client.JungleClient;
 import edu.colostate.cs.cs414.chesshireCoders.jungleUtil.game.User;
+import edu.colostate.cs.cs414.chesshireCoders.jungleUtil.requests.GetUserRequest;
 import edu.colostate.cs.cs414.chesshireCoders.jungleUtil.requests.UnRegisterRequest;
 import edu.colostate.cs.cs414.chesshireCoders.jungleUtil.responses.GetUserResponse;
+import edu.colostate.cs.cs414.chesshireCoders.jungleUtil.security.AuthToken;
 import javafx.application.Platform;
 
 public class AccountHandler extends Listener {
 
     private HomeController homeController;
 
-    private User userInfo;
+    private static User userInfo;
```

Comment:
In what other use case are we going to need to access the user's info on the client? It seems to me that we don't need a static variable for it.

Reply:
Yeah, its not needed. I changed it.